lib/kubeclient/common.rb

```diff
@@ -189,8 +189,8 @@ def define_entity_methods
         end
 
         # get a single entity of a specific type by name
-        define_singleton_method("get_#{entity.method_names[0]}") do |name, namespace = nil|
-          get_entity(klass, entity.resource_name, name, namespace)
+        define_singleton_method("get_#{entity.method_names[0]}") do |name, ns = nil, opts = {}|
+          get_entity(klass, entity.resource_name, name, ns, opts)
```

Comment:
@NickLaMuro the `namespace` => `ns` renaming is backward-compatible?

Reply:
For consistency with the existing functions, this could be kept as `namespace` (instead of `ns`).

Reply:
@simon3z @stefanmb This was done because of line length issues for rubocop.  Wasn't sure of a better way to do this, and I don't know how concerned you are with rubocop failures (but that did cause the travis run to fail without me doing this).